Dockerfile

```
@@ -0,0 +1,59 @@
+    qemu-kvm \
+    libvirt-clients \
+    libvirt-daemon-system \
+    bridge-utils \
+    sudo \
+    && rm -rf /var/lib/apt/lists/*
+
+ENV JAVA_HOME=/usr/lib/jvm/java-17-openjdk-amd64
+ENV PATH="$PATH:$JAVA_HOME/bin"
+
+ENV ANDROID_SDK_ROOT=/opt/android-sdk
+RUN mkdir -p ${ANDROID_SDK_ROOT}/cmdline-tools && \
```

Hadolint (2.14.0)

[warning] Use WORKDIR to switch to a directory

(DL3003)